model/itemTab.cc

```diff
@@ -244,35 +246,38 @@ namespace minsky
                         varAttribVals.push_back(to_string(v->sliderMax));
                         varAttribVals.push_back(to_string(v->value()));
                     
-                        for (auto& i:varAttrib) 
-                          {
-                            cairo_move_to(cairo,x,y-1.5*rowHeight);                    
-                            pango.setMarkup(i);
-                            pango.show();                  
-                            colWidth=std::max(colWidth,5+pango.width());  
-                            x+=colWidth;	
-                            colLeftMargin[iC].push_back(x);                        				    
-                          }
+                        if (it==itemVector[0]) {
+                          for (auto& i:varAttrib) 
+                            {
+                              cairo_move_to(cairo,x,y-1.5*rowHeight);                    
+                              pango.setMarkup(i);
+                              pango.show();                  
+                              colWidth=std::max(colWidth,5+pango.width());  
+                              x+=colWidth;	
+                              colLeftMargin[iC].push_back(x);                        				    
```

Comment:
Wouldn't this be a right margin?

Reply:
How is iC calculated?

Reply:
It's incremented at the end of loop (for auto& it: itemVector).

Reply:
> Wouldn't this be a right margin?

I think it is the left margin of a column. Now that I think about it, I may be able to use to adjust the column widths.